### PR TITLE
Basic things

### PR DESCRIPTION
**Pull request by antonmolodykh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Example of interaction (python):
+
+```
+from pydbus import SessionBus
+
+bus = SessionBus()
+browser_bus = bus.get('screenly.webview', '/Screenly')
+
+browser_bus.loadPage("www.example.com")
```
Comment (vpetersson) [inline README.md:54]: We need to specify the protocol here. Keep in mind that we need to support `http://`, `https://` and `rtsp://` and potentially more in the future.

File: src/main.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    QApplication::changeOverrideCursor(cursor);
+
+    MainWindow *window = new MainWindow();
+    window -> show();
+
+    QDBusConnection connection = QDBusConnection::sessionBus();
+
+    if (!connection.registerObject("/Screenly", window,  QDBusConnection::ExportAllSlots))
+    {
+        qWarning() << qPrintable("Can't register object");
```
Comment (sergey-borovkov) [inline src/main.cpp]: there is no point in using qPrintable here

File: src/main.cpp [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+int main(int argc, char *argv[])
+{
+    QApplication app(argc, argv);
+
+    QCursor cursor(Qt::BlankCursor);
+    QApplication::setOverrideCursor(cursor);
+    QApplication::changeOverrideCursor(cursor);
+
+    MainWindow *window = new MainWindow();
+    window -> show();
```
Comment (sergey-borovkov) [inline src/main.cpp]: don't add spaces between ->

**Comment by vpetersson** [comment]
Yay! Looking forward to first steps here.

**Pull request merged**